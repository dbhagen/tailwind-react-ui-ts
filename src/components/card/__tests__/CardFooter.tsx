import React from 'react'
import { shallow } from 'enzyme'

import { CardFooter } from '..'

const setup = (testProps = {}) => {
  const props = Object.assign(testProps)

  const wrapper = shallow(<CardFooter {...props}>Hello World!</CardFooter>)

  return {
    props,
    wrapper,
  }
}

describe('CardFooter', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup()

    expect(wrapper).toMatchSnapshot()
  })
})
