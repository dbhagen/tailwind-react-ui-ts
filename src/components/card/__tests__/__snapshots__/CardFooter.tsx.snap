// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardFooter renders matching snapshot 1`] = `
<Flex
  col={false}
  inline={false}
  inlineFlex={false}
  is="div"
  items="end"
  reverse={true}
  wrap={false}
  wrapReverse={false}
>
  <Component
    key=".0"
    rounded="none"
  />
</Flex>
`;
